package.json

```diff
@@ -18,19 +18,20 @@
   "types": "lib/peg.d.ts",
   "bin": "bin/peggy",
   "scripts": {
-    "pretest": "npm run parser",
-    "test": "npm run test:bare",
-    "test:bare": "mocha --recursive test --exclude test/browser.stub.js --reporter min",
+    "clean": "rimraf -f build && rimraf -f browser && mkdir build && cd build && mkdir ts && mkdir rollup && mkdir terser && cd .. && mkdir browser",
+    "terser": "terser -c passes=2 -m --sequences 40 build/rollup/peg.umd.js > build/terser/peggy.umd.min.js && terser -c passes=2 -m --sequences 40 --module build/rollup/peg.esmodule.js > build/terser/peggy.esmodule.min.js",
+    "rollup": "rollup -c",
+    "ts": "tsc --build tsconfig.json",
+    "set_version": "node -e \"const fs = require('fs'), pkg = JSON.parse(fs.readFileSync('./package.json')); fs.writeFileSync('./lib/version.js', '\\n\\'use strict\\';\\n\\nmodule.exports = \\'' + pkg.version + '\\';\\n');\"",
+    "test": "jest --coverage",
     "lint": "eslint . --ext .js bin/peggy",
-    "benchmark": "./benchmark/run",
-    "build": "npm run browser:build && npm run benchmark:build && npm run test:build",
-    "browser:clean": "rm -rf browser/",
-    "prebrowser:build": "npm run parser",
-    "browser:build": "browserify lib/peg.js -s peggy -o browser/peggy.min.js",
-    "postbrowser:build": "node tools/header browser/peggy.min.js && cp browser/peggy.min.js docs/vendor/peggy/peggy.min.js",
-    "benchmark:build": "browserify benchmark/browser.stub.js -o docs/js/benchmark-bundle.min.js && node tools/header docs/js/benchmark-bundle.min.js",
-    "test:build": "browserify test/browser.stub.js -o docs/js/test-bundle.min.js && node tools/header docs/js/test-bundle.min.js",
-    "parser": "node bin/peggy -o lib/parser.js --format commonjs src/parser.pegjs"
+    "benchmark": "node ./benchmark/run_bench.js",
+    "parser": "node bin/peggy -o lib/parser.js --format commonjs src/parser.pegjs",
+    "distro": "rm -f browser/peggy.min.js && cp build/terser/peggy.umd.min.js browser/peggy.min.js",
```

Comment:
`deploy` instead of `distro`?

Reply:
it doesn't do any deploying, though?  i would expect that to push to an active server

Reply:
i switched

for the record, in my head, this meant "create the distribution"